apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -247,14 +462,91 @@ export default class ExportHTMLModal extends Vue {
         StatusCountModule.countOf(this.filter, 'PassingTestsFailedControl') +
         StatusCountModule.countOf(this.filter, 'FailedTests')
     };
+    this.outputData.severity = {
+      none: SeverityCountModule.none(this.filter),
+      low: SeverityCountModule.low(this.filter),
+      medium: SeverityCountModule.medium(this.filter),
+      high: SeverityCountModule.high(this.filter),
+      critical: SeverityCountModule.critical(this.filter)
+    };
     this.outputData.files = [];
     this.outputData.controlSets = [];
   }
 
+  // Replace all found illegal characters in string with compliant string equivalent
+  replaceIllegalChar(text: string): string {
+    for (const illegalChar in ILLEGAL_CHAR_SET) {
```

Comment:
i think if you swapped over to using for..of instead of for..in, you should be able to not need the type cast stuff going on in what's currently line 481

Reply:
Changed as requested.